torch_geometric/explain/algorithm/dummyexplainer.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+
+import torch
+
+from torch_geometric.data import Data
+from torch_geometric.explain.base import ExplainerAlgorithm
+from torch_geometric.explain.explanations import Explanation
+
+
+class RandomExplainer(ExplainerAlgorithm):
```

Comment:
Maybe this would be better defined in the tests? I can't see a reason users would want to use this?

Reply:
Maybe for benchmarking? if someone proposes a new explanation algorithms, it can be compared to the dummy explainers (in the same spirit as a dummy model from `sklearn`).